src/solver/hydro/monthly/h2o_m_sys.h

```diff
@@ -35,5 +35,6 @@
 #include <string.h>
 #include <memory.h>
 #include <time.h>
+#include <vector>
```

Comment:
Try to include only necessary headers. You may include <vector> in the .cpp implementation file instead.